components/molecule/modal/src/index.scss

```diff
@@ -1,31 +1,80 @@
 @import '~@schibstedspain/sui-theme/lib/index';
 @import './settings';
 
+@mixin modal-position {
```

Comment:
Since this modal-position is something static and is not changing, I would suggest to change it from `@mixin` to `%placeholder` due to it reduces the final `css bundle` or on the other hand, `@mixin` duplicates the `css`